src/datasets/load.py

```diff
@@ -1541,7 +1541,8 @@ def load_dataset(
     features: Optional[Features] = None,
     download_config: Optional[DownloadConfig] = None,
     download_mode: Optional[DownloadMode] = None,
-    ignore_verifications: bool = False,
+    verification_mode: Optional[VerificationMode] = None,
```

Comment:
can we set `Optional[Union[VerificationMode,str]]` as a type annotation here (and same for `DownloadMode`)? i'm tired of my IDE shouting at me when I provide string as a `download_mode`.